### ISSUE TITLE
upgrade to v1.25.2 did not work - LOTS of "teslamate_1  | (no logger present) unexpected logger message" errors

### ISSUE THREAD

**Issue by JonMurphy** [adriankumpf/teslamate#2320]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What happened?

Today's (2022-01-14) upgrade to v1.25.2 did not work.  Caused my RPi3B+ (buster) with Docker (version 20.10.12, build e91ed57) to run very hot and slow way down.  

Needed to un-plug power.

### Expected Behavior

No errors and no massive slow-down.

### Steps To Reproduce

1. did backup:
```
cd teslamate
docker-compose exec -T database pg_dump -U teslamate teslamate > /home/pi/teslamate/$(date "+%Y-%m-%d_at_%H.%M.%S")-teslamate.bck
cp docker-compose.yml /home/pi/teslamate/$(date "+%Y-%m-%d_at_%H.%M.%S")-docker-compose.yml
```

2 .then did `pull`
```
docker-compose pull
docker-compose up -d
```

3. Could not do anything with Teslamate.  Acted like RPi had frozen.  Power down by removing power & then power up
4. login but everything was very slow
4. looked at `top` and saw `beam.smp` was running at 266 %CPU.
5. did `docker-compose down`
6. did `docker-compose up` and saw errors in Log Output below.

7. reverted back to v1.25.0

### Relevant log output

```shell
got these errors after power-up:

pi@teslamate:~/teslamate $ docker-compose up
Creating network "teslamate_default" with the default driver
Creating teslamate_database_1  ... done
Creating teslamate_teslamate_1 ... done
Creating teslamate_mosquitto_1 ... done
Creating teslamate_grafana_1   ... done
Attaching to teslamate_database_1, teslamate_teslamate_1, teslamate_mosquitto_1, teslamate_grafana_1
teslamate_1  | waiting for postgres at database:5432
teslamate_1  | sleep: cannot read realtime clock: Operation not permitted
mosquitto_1  | 0: mosquitto version 2.0.14 starting
mosquitto_1  | 0: Config loaded from /mosquitto-no-auth.conf.
mosquitto_1  | 0: Opening ipv4 listen socket on port 1883.
mosquitto_1  | 0: Opening ipv6 listen socket on port 1883.
mosquitto_1  | 0: mosquitto version 2.0.14 running
database_1   | 
database_1   | PostgreSQL Database directory appears to contain a database; Skipping initialization
database_1   | 
database_1   | 2022-01-14 22:30:44.138 UTC [1] LOG:  starting PostgreSQL 12.9 (Debian 12.9-1.pgdg110+1) on arm-unknown-linux-gnueabihf, compiled by gcc (Debian 10.2.1-6) 10.2.1 20210110, 32-bit
database_1   | 2022-01-14 22:30:44.162 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
database_1   | 2022-01-14 22:30:44.162 UTC [1] LOG:  listening on IPv6 address "::", port 5432
database_1   | 2022-01-14 22:30:46.312 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
database_1   | 2022-01-14 22:30:48.220 UTC [27] LOG:  database system was shut down at 2022-01-14 22:23:10 UTC
database_1   | 2022-01-14 22:30:50.521 UTC [1] LOG:  database system is ready to accept connections
teslamate_1  | (no logger present) unexpected logger message: {log,error,"~s~n",["erts_poll_wait() failed: eperm (1)\n"],#{error_logger=>#{emulator=>true,tag=>error},time=>1642199453595542}}
teslamate_1  | (no logger present) unexpected logger message: {log,error,"~s~n",["erts_poll_wait() failed: eperm (1)\n"],#{error_logger=>#{emulator=>true,tag=>error},time=>1642199453595576}}
teslamate_1  | (no logger present) unexpected logger message: {log,error,"~s~n",["erts_poll_wait() failed: eperm (1)\n"],#{error_logger=>#{emulator=>true,tag=>error},time=>1642199453595588}}
. . .
teslamate_1  | (no logger present) unexpected logger message: {log,error,"~s~n",["erts_poll_wait() failed: eperm (1)\n"],#{error_logger=>#{emulator=>true,tag=>error},time=>1642199453603481}}
teslamate_1  | (no logger present) unexpected logger message: {log,error,"~s~n",["erts_poll_wait() failed: eperm (1)\n"],#{error_logger=>#{emulator=>true,tag=>error},time=>1642199453603492}}
grafana_1    | t=2022-01-14T22:30:54+0000 lvl=info msg="Starting plugin search" logger=plugins
teslamate_1  | =ERROR REPORT==== 14-Jan-2022::22:30:53.603504 ===
teslamate_1  | erts_poll_wait() failed: eperm (1)
teslamate_1  | 
teslamate_1  | 
teslamate_1  | =ERROR REPORT==== 14-Jan-2022::22:30:53.603515 ===
teslamate_1  | erts_poll_wait() failed: eperm (1)
teslamate_1  | 
...
```
```


### Screenshots

_No response_

### Additional data

_No response_

### Type of installation

Docker

### Version

v1.25.0

**Comment by JonMurphy** [comment]
I did see a tentative fix here:
https://github.com/adriankumpf/teslamate/issues/2302#issuecomment-1011362938
but have not tried it...

**Comment by natrlhy** [comment]
After seeing all the issues and letting the dust settle, I went ahead and upgraded my Raspberry Pi 3 Model B Rev 1.2 from buster to bullseye by following [this guide](https://forums.raspberrypi.com/viewtopic.php?t=323279). Also @adriankumpf mentioned his builds are all based on running bullseye so I figured may as well go all in…
After the buster to bullseye upgrade, I followed the standard process to upgrade teslamate.

`docker-compose` seemed like it got removed during the upgrade process, so I simply ran `pip3 install docker-compose` and then ran my standard commands to upgrade teslamate. No issues and happily running 1.25.2.

```
sudo -i 
cd ~/docker
docker-compose down
docker-compose pull
docker-compose up -d
```

Thanks for this conversation and working through everything!

**Comment by adriankumpf** [comment]
> I did see a tentative fix here: [#2300 (comment)](https://github.com/adriankumpf/teslamate/issues/2300#issuecomment-1010905276) -and here- [#2302 (comment)](https://github.com/adriankumpf/teslamate/issues/2302#issuecomment-1011362938) but have not tried it...

Please try these fixes.

**Comment by JonMurphy** [comment]
This works very well!  👍   See: https://github.com/adriankumpf/teslamate/issues/2302#issuecomment-1011362938

<br>

> So I do not want to solicit this as an "official" fix, but . . .
>
> 
> ```
> # Get signing keys to verify the new packages, otherwise they will not install
> rpi ~$ sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 04EE7237B7D453EC 648ACFD622F3D138
> # Add the Buster backport repository to apt sources.list
> rpi ~$ echo 'deb http://httpredir.debian.org/debian buster-backports main contrib non-free' | sudo tee -a /etc/apt/sources.list.d/debian-backports.list
> rpi ~$ sudo apt update
> rpi ~$ sudo apt install libseccomp2 -t buster-backports
> ```
> 
> I was able to follow that and got 1.25.2 working.
> 

v 1.25.2 works for me as well!
